core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala

```diff
@@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
 import org.apache.spark.SecurityManager
 import org.apache.spark.metrics.MetricsSystem
 
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
```

Comment:
ok. I will push more commits and update pull request.
Thanks.

On Tue, Apr 26, 2016 at 2:52 PM, Sean Owen <notifications@github.com> wrote:

> In core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala
> <https://github.com/apache/spark/pull/12697#discussion_r61053629>:
>
> > @@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
> >  import org.apache.spark.SecurityManager
> >  import org.apache.spark.metrics.MetricsSystem
> >
> > +import org.slf4j.Logger
> > +import org.slf4j.LoggerFactory
>
> No you simply push more commits to this branch
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12697/files/5a437a12fb0c6cfdd7119d63bebc3f3f2c935d5f#r61053629>
>



-- 
Mihir Monani
(+91)-9429473434